Src/WitsmlExplorer.Api/Services/WellService.cs

```diff
@@ -65,21 +68,21 @@ private async Task<IEnumerable<Well>> GetWellsInformation(string wellUid = null)
                     ItemState = well.CommonData.ItemState,
                     StatusWell = well.StatusWell,
                     PurposeWell = well.PurposeWell,
-                    WellDatum = WellDatum.FromWitsmlWellDatum(well.WellDatum),
+                    WellDatum = WellDatum.FromWitsmlWellDatum(well.WellDatum).FirstOrDefault(),
```

Comment:
It makes little sense to return only the first WellDatum or WellLocation when the underlying WITSML model contains a list. The API Well model should either contain lists of WellDatums and WellLocations or not contain them at all. I think that we should remove them from the API model given that we have no use for them in the frontend or any of the workers.

Reply:
Also every time one enters the production server, thousands of wells are fetched, so it would be good to keep the return values of GetWellsInformation as small as possible.

Reply:
The amount of data where not changed, it was processed in the same way, just mapping was done only for first item. Now all received data in list are mapped, but because of current DTO sent to frontend contains one item only, the first item is used - as it was so, but it was done in the static method already, just not so obvioous